### PR TITLE
Palo Alto minemield integration

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Integrations/integration-Minemeld.yml [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+        comment = demisto.args().get('comment','')
+        share_level = demisto.args().get('share_level','red')
+
+        if demisto.command() == 'add-to-miner':
+            action = ADD_ACTION
+        elif demisto.command() == 'remove-from-miner':
+            action = REMOVE_ACTION
+
+        preform_action(MM, miner, type_, value, action, comment, share_level)
+        demisto.results('Preformed action successfully')
```
Comment (NeelRust) [inline Integrations/integration-Minemeld.yml]: Typo here - `Performed`

File: Integrations/integration-Minemeld.yml [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+  type: python
+  commands:
+  - name: add-to-miner
```
Comment (NeelRust) [inline Integrations/integration-Minemeld.yml]: command names dont have product name - needs a bit of design review

File: Integrations/integration-Minemeld.yml [hunk 3]
```diff
@@ -0,0 +1,286 @@
+commonfields:
+  id: Palo Alto - Minemeld
+  version: -1
+name: Palo Alto - Minemeld
+display: Palo Alto - Minemeld
```
Comment (NeelRust) [inline Integrations/integration-Minemeld.yml]: Name: `Palo Alto Minemeld`

File: Integrations/integration-Minemeld.yml [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+
+        preform_action(MM, miner, type_, value, action, comment, share_level)
+        demisto.results('Preformed action successfully')
+  type: python
+  commands:
+  - name: add-to-miner
+    arguments:
+    - name: miner
+      required: true
+      description: Miner name.
```
Comment (NeelRust) [inline Integrations/integration-Minemeld.yml]: explain where you can get this information from Minemeld UI

File: Integrations/integration-Minemeld.yml [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+      - white
+      - green
+      - amber
+      - red
+      defaultValue: red
+      description: Comment to the value.
+    description: The share-level of the value.
+  - name: remove-from-miner
+    arguments:
+    - name: miner
```
Comment (NeelRust) [inline Integrations/integration-Minemeld.yml]: Explain where to get this information from Minemeld UI

**Comment by erezh31** [comment]
@NeelRust fixes pushed

File: Tests/conf.json [hunk 6]
```diff
@@ -471,6 +471,10 @@
         {
             "integrations": "TruSTAR",
             "playbookID": "TruSTAR Test"
+        },
+        {
+            "integrations": "MineMeld",
+            "playbookID": "minemeld_test"
```
Comment (erezh31) [inline Tests/conf.json]: It's the wrong place.. notice that you under "skipped" key on the JSON. plz add it under "tests" key (first part of the JSON)

**Pull request merged**